src/test/java/com/company/verifytests/UserInfoTest.java

```diff
@@ -0,0 +1,52 @@
+package com.company.verifytests;
+
+import org.testng.annotations.Test;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.concurrent.TimeUnit;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class UserInfoTest extends BaseTest {
+    LoginPage loginPage = new LoginPage(driver);
+    LocalDate date = LocalDate.now();
+    DateTimeFormatter format =
+            DateTimeFormatter.ofPattern("yyyy-MM-dd");
+    String currentDate = date.format(format);
+
+    @Test
+    public void testUserInfo(){
+        loginPage = openLoginPage();
+        loginPage.loginIntoSystem("TestUser07","Vfylhfujhf!1");
+        
+        MainMenuPage mainPage = new MainMenuPage(driver);
+        mainPage.clickMyInfo();
+        MyInfoPage info = new MyInfoPage(driver);
+        info.clickEditSave();
+        info.inputFirstName("Olya");
+        info.inputLastName("Bilynska");
+        info.inputEmployeeId("1");
+        info.inputOtherId("07");
+        info.inputDriverNo("78956");
+        info.chooseGender();
+        info.chooseNationality();
+        info.chooseMaritalStatus();
+        info.inputDateOfBirth("2000-08-11");
+        info.clickEditSave();
+        mainPage.moveToTime()
```

Comment:
You can wrap editting of user data into single method, as it is like flow that can be reused by other developer or test.

Reply:
Fixed

Reply:
Don't clearly understand. Do I need to wrap chooseMaritalStatus() .inputDateOfBirth() .clickEditSave() into a single method?

Reply:
Done